### PR TITLE
Move Schemifier call to the bottom of boot sequence

### PR DESCRIPTION
**Pull request by heharkon**
Antonio suggested that moving the Schemifier call later in the boot sequence would give better error response through the browser, if DB init fails. I tried it and it seems to work better like this.

### CODE REVIEW

**Review by farmdawgnation — APPROVED** [review]
Seems reasonable, do any of these changes need to be made to lift/blank-app.g8?

**Comment by heharkon** [comment]
Blank app is safe to leave untouched, as it doesn't have DB setup.

**Comment by farmdawgnation** [comment]
Works for me.

File: src/main/g8/src/main/webapp/templates-hidden/default.html [hunk 1]
```diff
@@ -1,44 +1,23 @@
...
+  <title>App: </title>
+  <link href="/assets/css/app.css" rel="stylesheet" type="text/css" />
+  <script id="jquery" src="/classpath/jquery.js" type="text/javascript"></script>
+  <script id="json" src="/classpath/json.js" type="text/javascript"></script>
+</head>
 
-      <div class="column span-6 colborder sidebar">
-        <hr class="space" >
+<body>
+  <div>
```
Comment (Shadowfiend) [inline src/main/g8/src/main/webapp/templates-hidden/default.html]: Any reason we redid all the HTML here?

Comment (heharkon) [reply src/main/g8/src/main/webapp/templates-hidden/default.html]: Oops! This commit should be reverted! I don't know how this kind of change ended up here! :-o Good catch.

Comment (heharkon) [reply src/main/g8/src/main/webapp/templates-hidden/default.html]: I reverted that commit. Now I'll erase my branch to not accidentally screw up things again.

Comment (heharkon) [reply src/main/g8/src/main/webapp/templates-hidden/default.html]: @farmdawgnation Now I think I know how that extra commit was there in begin with. I had this repo cloned earlier for my own purposes, and didn't realize it while I cloned already and started to make changes assuming it was a clean clone. I'm sorry! But I think you can now merge this latest change so the template is reverted as it was originally.

**Pull request merged**